.github/workflows/client.yaml

```
@@ -42,6 +42,37 @@ jobs:
       - name: Validate
         run: cd client && pnpm validate
 
+  coverage:
+    name: Coverage
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: PNPM Store Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.pnpm-store
+          key: ${{ runner.OS }}-pnpm-${{ hashFiles('**/pnpm-lock.yaml') }}
```

Cache collision here - set this to use something like `-pnpm-coverage-` for the cache key.

---

Aha, good catch - thanks

```suggestion
          key: ${{ runner.OS }}-pnpm-coverage-${{ hashFiles('**/pnpm-lock.yaml') }}
```

---

That's a shame, honestly - I would've hoped that the cache used for validation could also be used for coverage. I'm sure GitHub's servers can handle it lmao